linker/shared/src/main/scala/org/scalajs/linker/frontend/optimizer/OptimizerCore.scala

```diff
@@ -610,6 +610,9 @@ private[optimizer] abstract class OptimizerCore(
           pretransformExpr(tree)(finishTransform(isStat))
         }
 
+      case prop @ LinkTimeProperty(name) =>
+        config.coreSpec.linkTimeProperties.transformLinkTimeProperty(prop)
```

Comment:
You will also have to deal with them in `pretransformExpr`, so that they become literals during pre-transformation. Otherwise constant-folding will not work.